test-engine-core-modules/src/rdsserializer/rdsserializer.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+import os
+from typing import Any
+
+from test_engine_core.interfaces.iserializer import ISerializer
+from test_engine_core.plugins.enums.plugin_type import PluginType
+from test_engine_core.plugins.enums.serializer_plugin_type import SerializerPluginType
+from test_engine_core.plugins.metadata.plugin_metadata import PluginMetadata
+
+
+# NOTE: Do not change the class name, else the plugin cannot be read by the system
+class Plugin(ISerializer):
+    """
+    The Plugin(rdsserializer) class specifies methods on serialization.
+    """
+
+    # Some information on plugin
+    _name: str = "rdsserializer"
+    _description: str = "rdsserializer supports deserializing R Data Serialization"
+    _version: str = "0.9.0"
```

Comment:
just wondering if within the project the version is kept consistent.. so 0.9.0 in this case... since this serializer is new a more logical version would be 0.1.0

Reply:
Yeah this would be a question for @imda-benedictlee or his colleauges, how do you keep track of the version of the test-engine-core and how would you update this? As also with the openapiconnect the test-engine-core have been changed, but the version wasn't bumped to my better knowledge

Reply:
Hi @laurensWe @uittenbroekrobbert sorry for getting to this late, we were deliberating how these core modules should be version controlled. 

Our current practice is to only bump the version number if there are any patches/ new features for that particular component (this does not cover the addition of new subcomponents). 

At our initial open source release, we started all components as v0.9.0 to follow the toolkit's release versioning as prior to this, there was no plugin architecture. The openapiconnector that was released subsequently was also given 0.9.0 as 0.9.0 is now taken to be the 'initial release'. 

Hence to keep our initial versioning of the core-modules subcomponents consistent, we would propose to have rdsserializer versioned as 0.9.0, without updating the test-engine-core-modules/test-engine-core versions.

Please let me know if this makes sense, happy to discuss further :)

Reply:
Ah yeah that makes perfect sense :)